### DISCUSSION TITLE
Unable to access set() and get() on a component after upgrading from 20.4 to 21.4

### DISCUSSION THREAD

**Discussion by elfalem** [GrapesJS/grapesjs#5311] [Q&A]
For example, the following code 
```
 component.set('resizable', true)
```
results in the this type error:
```
Property 'set' does not exist on type 'Component'
```

The release notes for 21.1 mention breaking changes to types/interfaces. However, I wasn't able to spot relevant changes in docs or commits. Has the interface for setting and getting properties on components changed? If so, what's the new syntax?

**Comment by artf** [comment]
Did you try to restart the TS server, post-library upgrade? 
Looks like you're missing `@types/backbone` but it should be already there as a [dependency](https://github.com/GrapesJS/grapesjs/blob/4f0c5b1bea583bc2ffaca491098e11c58b343620/package.json#L20).

**Comment by elfalem** [comment]
Restarting the TS server and trying on another machine didn't help. The app is using Angular, so perhaps it's related to use of Backbone in Angular.

**Reply by artf** [reply]
did you check `@types/backbone`?

**Reply by elfalem** [reply]
Yes. It's there. It seems like adding the compile option `esModuleInterop: true` in tsconfig and upgrading the typescript version solved it.
